src/locales/locale-en.json

```diff
@@ -563,6 +573,71 @@
     "WHAT": "What does it mean to be inactive?",
     "HELP": "Everyone who did not log into the group for {dayCount} days or more counts as inactive. These users will not receive notification emails anymore."
   },
+  "CONFLICT": {
+    "SETUP_HEADER": "Is there a conflict with {user}?",
+    "SETUP": "Conflict resolution setup",
+    "STATEMENT": "Statement",
+    "STEPPER1a": "You are about to start a group-wide conflict resolution process!",
+    "STEPPER1b": "This will involve all of your group's members with editing permissions to try and find the best way to proceed regarding this conflict.",
+    "STEPPER2a": "Thank you for caring for your community!",
+    "STEPPER2b": "It is important to openly address conflicts and you seem to know this. Thank you for caring enough to not shy away from the unpleasant task of surfacing a social issue!",
+    "STEPPER3a": "Here's what will happen:",
+    "STEPPER3b": "This process will start an open chat and a vote.",
+    "STEPPER3c": "Every message in the chat will be visible for every member in the group - including the person this process is about!",
+    "STEPPER3d": "The voting is anonymous.",
+    "STEPPER3e": "It will go on for {votingDuration} days.",
+    "STEPPER3f": "During this time all voters can change their minds and vote differently as many times as they want.",
+    "STEPPER4a": "Please make an initial statement for the open group chat!",
+    "STEPPER4b": "Tell your group what made you think this process is justified and needed.",
+    "INITIAL": "Information",
+    "WITH": "This process is to resolve a conflict with {userName}",
+    "VOTING": {
+      "HEADLINE": "How do you want to proceed with the ongoing conflict with {userName}?"
+    },
+    "REQUIREMENTS": {
+      "ACTIVE_EDITORS": "Your group has {activeEditorsCount} active users with editing permissions. You need {requiredCount} to start a conflict resolution process.",
+      "NEWCOMER": "Before you can start a conflict resolution process, you need to receive more trust and gain editing permissions. In the meantime, please ask a fellow group member to start the process for you."
+    }
+  },
+  "ISSUE": {
+    "CREATION": {
+      "TOAST": "You successfully started a conflict resolution process"
+    },
+    "TITLE": "Issues",
+    "INITIATED_BY": "Initiated by {initiatorName}",
+    "PAST": "Past issues",
+    "PREVIOUS_VOTINGS": "Previous votings in this issue",
+    "PARTICIPANTS": "{count} members have been added to this discussion.",
+    "VOTING": {
+      "TIME_LEFT": "Time left to vote",
+      "BTN_START": "Vote now",
+      "BTN_DELETE": "Delete vote",
+      "TOAST": "Your vote was successfully submitted",
+      "TOAST_DELETE": "Your vote was deleted",
+      "NO_CHANGE": "{userName} stays in {groupName}",
```

Comment:
I would wonder about prefixing this with _No change:_ as it's a kind of _special_ status quo option.